cmd/talosctl/cmd/mgmt/gen/config.go

```diff
@@ -110,10 +110,10 @@ func fixControlPlaneEndpoint(u *url.URL) *url.URL {
 	return u
 }
 
-// GenV1Alpha1Config generates the Talos config bundle
+// genV1Alpha1Config generates the Talos config bundle
 //
-// GenV1Alpha1Config is useful for integration with external tooling options.
-func GenV1Alpha1Config(genOptions []generate.GenOption,
```

Comment:
This function was made public on purpose in one of the previous PRs. Even if that is not the perfect place for it, let's not break the code which imports it

Reply:
https://github.com/talos-systems/talos/pull/3650